event-service/src/main/java/hcmus/alumni/event/controller/EventServiceController.java

```diff
@@ -108,15 +112,16 @@ public class EventServiceController {
 	
 	@GetMapping("")
 	public ResponseEntity<HashMap<String, Object>> getEvents(
-	        @RequestParam(value = "page", required = false, defaultValue = "0") int page,
-	        @RequestParam(value = "pageSize", required = false, defaultValue = "10") int pageSize,
-	        @RequestParam(value = "title", required = false, defaultValue = "") String title,
-	        @RequestParam(value = "orderBy", required = false, defaultValue = "organizationTime") String orderBy,
-	        @RequestParam(value = "order", required = false, defaultValue = "desc") String order,
-	        @RequestParam(value = "facultyId", required = false) Integer facultyId,
-	        @RequestParam(value = "tagNames", required = false) List<String> tagNames,
-	        @RequestParam(value = "statusId", required = false) Integer statusId,
-	        @RequestParam(value = "mode", required = false, defaultValue = "1") int mode) {
+			@RequestHeader("userId") String userId,
```

Comment:
```suggestion
			@RequestHeader(value = "userId", defaultValue = "") String userId,
```
thêm defaulValue để userId ko null trong trường hợp gọi api ko có token